internal/system/system_test.go

```
@@ -27,3 +27,30 @@ func TestSystemGetTime(t *testing.T) {
 		t.Errorf("GetTime time is not before current time, return val: %s", rw.Body.String())
 	}
 }
+
+func TestSystemGetVersion(t *testing.T) {
+	req, err := http.NewRequest("GET", "/version", nil)
```

golangci-lint (2.12.1)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)